api/GL/glcorearb.h

```diff
@@ -3958,6 +3958,20 @@ GLAPI void APIENTRY glMaxShaderCompilerThreadsKHR (GLuint count);
 #define GL_KHR_texture_compression_astc_sliced_3d 1
 #endif /* GL_KHR_texture_compression_astc_sliced_3d */
 
+#ifndef GL_AMD_framebuffer_multisample_advanced
+#define GL_AMD_framebuffer_multisample_advanced
+#define GL_RENDERBUFFER_STORAGE_SAMPLES_AMD 0x91B2
+#define GL_MAX_COLOR_FRAMEBUFFER_SAMPLES_AMD 0x91B3
```

Comment:
I'm confused by why you have 0x91B3-7 here, but 0x92B3-7 in gl.xml. These header are supposed to be regenerated from gl.xml, and I'm pretty certain you did not do that. In any event, it's better to leave the generated header updates out of extension submissions, since doing this tends to generate conflicts with other PRs in flight.